packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -203,41 +261,45 @@ class InputDateTimePicker extends React.Component {
 				onFocus={this.onFocus}
 				onBlur={this.onBlur}
 			>
-				<DebounceInput
-					{...inputProps}
-					inputRef={ref => {
-						this.inputRef = ref;
-					}}
-					type="text"
-					placeholder={getFullDateFormat(this.getDateOptions())}
-					value={this.state.textInput}
-					debounceTimeout={300}
-					onChange={this.onInputChange}
-					className="form-control"
-					autoComplete="off"
-				/>
-				<div
-					className={theme['dropdown-wrapper']}
-					ref={ref => {
-						this.dropdownWrapperRef = ref;
-					}}
-				>
-					<Overlay container={this.dropdownWrapperRef} show={this.state.showPicker}>
-						<Popover className={theme.popover} id={this.popoverId}>
-							<DateTimePicker
-								manageFocus
-								selection={{
-									date: this.state.date,
-									time: this.state.time,
-								}}
-								onSubmit={this.onPickerChange}
-								useTime={this.props.useTime}
-								useSeconds={this.props.useSeconds}
-								useUTC={this.props.useUTC}
-							/>
-						</Popover>
-					</Overlay>
-				</div>
+				<form onSubmit={this.onSubmit}>
```

Comment:
This should be here only if we are in formMode.
Otherwise we can have a form in a form if DateTimePicker is used in a form